lazy_loader/__init__.py

```diff
@@ -269,12 +267,9 @@ def _check_requirement(require: str) -> bool:
         True if the installed version of the dependency matches
         the specified version, False otherwise.
     """
-    import packaging.requirements
+    import importlib.metadata as importlib_metadata
```

Comment:
Does it make sense to drop the `as importlib_metadata` and `s/_/./` on L276?

```suggestion
    import importlib.metadata
```

Leaving it as-is seems fine, too, as that reduces the scope of change if for some reason we need a backported `importlib-metadata` again.

Reply:
See 799f07e.